### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Bump dependencies and port small changes from other PRs.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/clientconn/conn.go [hunk 1]
```diff
@@ -581,26 +581,31 @@ func (c *conn) handleOpMsg(ctx context.Context, msg *wire.OpMsg, command string)
...
+			ok = v == 1
 		}
+
+		if !ok {
+			level = zap.WarnLevel
+		}
+	}
+
+	if closeConn {
+		level = zap.ErrorLevel
```
Comment (chilagrow) [inline internal/clientconn/conn.go:608]: Is it intended to change log level to errorLevel for `closeConn == true` even if `ok == 1`?

**Review by chilagrow — APPROVED** [review]
Just one question, thank you for updating dependencies :hugs:

File: internal/clientconn/conn.go (hunk 1 above)
Comment (AlekSi) [reply internal/clientconn/conn.go:608]: I don't think that case exists: `closeConn` represents connection closing as a result of handling some command, and that's always something unrecoverable

Comment (chilagrow) [reply internal/clientconn/conn.go:608]: I see, so it would never be ok == 1 that case. Thanks for the explanation :hugs:

**Pull request merged**